ui/components/ConfirmationModal.js

```diff
@@ -284,18 +284,24 @@ function ConfirmationMsg(props) {
             <Tab
               data-cy="validate-btn-modal"
               className={classes.tab}
-              label={<div style={{ display : "flex" }}> <DoneIcon style={{ margin : "2px" }}  fontSize="small"/><span className={classes.tabLabel}>Validate</span> </div>
+              label={<div style={{ display : "flex" }}
+                onClick={(e) => handleTabValChange(e,0)}
```

Comment:
@partik03 Don't use onClick under the label, this will create an intuitive click issue. As I tested this PR, I'm only able to change the table when exactly clicked on the text(label) not on the tab.

Reply:
Done